_sass/base/components/_picc-slider.scss

```diff
@@ -0,0 +1,60 @@
+picc-slider {
+  $handle-size: 24px;
```

Comment:
In my efforts to make my CSS more modular and maintainable, I want to be more strict about when I choose `px` versus when I choose `em`. Are these sizes that should change with browser zoom? If yes, they should be `em`.